frontend/resourcebundles/src/main/resources/GitMacheteBundle.properties

```diff
@@ -441,3 +441,41 @@ action.GitMachete.CheckoutPreviousAction.undefined.branch-name=No branches befor
 action.GitMachete.CheckoutParentAction.text=Git Machete: Checkout Parent Branch
 action.GitMachete.CheckoutParentAction.description=Checkout parent branch (''{0}'')
 action.GitMachete.CheckoutParentAction.undefined.branch-name=Branch (''{0}'') does not have a parent in the branch layout
+
+# Traverse
+
+action.GitMachete.TraverseRepositoryAction.description.text=Starting To Traverse
+action.GitMachete.TraverseRepositoryAction.description.empty-layout=Cannot traverse an empty branch layout
+action.GitMachete.TraverseRepositoryAction.GitMacheteToolbar.text=Traverse
+action.GitMachete.TraverseRepositoryAction.invoked-action.id=GitMachete.{0}
+
+action.GitMachete.TraverseRepositoryAction.dialog.traverse-approval.title=Traverse
+action.GitMachete.TraverseRepositoryAction.dialog.traverse-approval.text.HTML=<html>Do you want to traverse the branches of this layout?</html>
+action.GitMachete.TraverseRepositoryAction.dialog.traverse-approval.ok-text=traverse
+
+action.GitMachete.TraverseRepositoryAction.dialog.do-not-show-again=Don't show it again
+
+
+action.GitMachete.TraverseRepositoryAction.dialog.diverged-from-remote.text.HTML=<html><p>The branch <b>{0}</b> is divereged from remote <b>{1}</b>, and it is <b>{2}</b> remote.</p><p>Please choose the preferred action for syncing it:</p></html>
+action.GitMachete.TraverseRepositoryAction.dialog.diverged-from-remote.title=Resolve Divergence from Remote
+action.GitMachete.TraverseRepositoryAction.dialog.diverged-from-remote.newer-than-remote=older than
+action.GitMachete.TraverseRepositoryAction.dialog.diverged-from-remote.older-than-remote=newer than
+action.GitMachete.TraverseRepositoryAction.dialog.diverged-from-remote.rebase-on-remote=Rebase local branch on remote
+action.GitMachete.TraverseRepositoryAction.dialog.diverged-from-remote.merge-remote=Merge remote branch into local
+action.GitMachete.TraverseRepositoryAction.dialog.diverged-from-remote.force-push=Force push to remote
+action.GitMachete.TraverseRepositoryAction.dialog.diverged-from-remote.reset-on-remote=Reset local on remote
+
+action.GitMachete.TraverseRepositoryAction.dialog.diverged-from-parent.title=Resolve Divergence from Parent
+action.GitMachete.TraverseRepositoryAction.dialog.diverged-from-parent.text.HTML=<html><p>The branch <b>{0}</b> is diverged from parent <b>{1}</b>.</p><p>Please choose the preferred action for resolving it:</p></html>
+action.GitMachete.TraverseRepositoryAction.dialog.diverged-from-parent.rebase-on-parent=Rebase branch on the parent
+action.GitMachete.TraverseRepositoryAction.dialog.diverged-from-parent.reset-to-parent=Reset branch to the parent
+action.GitMachete.TraverseRepositoryAction.dialog.diverged-from-parent.merge-parent-into-current=Merge parent into current
+action.GitMachete.TraverseRepositoryAction.dialog.diverged-from-parent.merge-parent-into-current.comment=Warning: this is discouraged
```

Comment:
why? (i know why - but the explanation should be given here)

Reply:
(suggested to remove completely, won't be problem anymore :trollface:)